vkbeautify/vkbeautify-tests.ts

```diff
@@ -0,0 +1,119 @@
+import VKBeautify from 'vkbeautify';
```

Comment:
Ok, thanks.

Somehow I perceive it as _When I import this module, als those functions will be declared globally_. :)

But so no namespace would be preferable then? I mean, preferable over the `export = `?